sparse-11/var/lib/environment/nemo/99-qtscenegraph.conf

```diff
@@ -0,0 +1,2 @@
+# This flag needs to be set, regardless of its value
+QT_OPENGL_NO_BGRA=1
```

Comment:
have you checked if this is still needed?

Reply:
Based on quick testing this wasn't needed but should be tested more before removing.